apiary kafka metastore listener (#68)
ExpediaGroup/apiary-metastore-docker: CHANGELOG.md

The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).

## [1.8.0] - 2020-02-06

> 

Reviewer: ```suggestion
```

> ### Changed

Reviewer: ```suggestion
### Added
```

> - Add optional Apiary Kafka metastore listener, to send Hive metadata events to Kafka topic.

Reviewer: ```suggestion
- Added an optional Apiary metastore listener which can be used to send Hive metadata events to a Kafka topic.
```
